essentials-framework/Essentials Core/PepperDashEssentialsBase/JoinMaps/JoinMapBase.cs

```diff
@@ -363,8 +364,12 @@ public class JoinMetadata
         /// <summary>
         /// A label for the join to better describe it's usage
         /// </summary>
-        [JsonProperty("label")]
-        public string Label { get; set; }
+        [Obsolete]

```

Comment:
For now, a better way to do this would be to have private string field called _description that both Label and Description get/set. Also, Description is missing it's XML Comments.

Reply:
Also, add a message to the Obsolete attribute indicating which property to use instead.

Reply:
Should be fixed.